service.subtitles.napisy24pl/service.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
+from urllib.parse import unquote
+
+try:
+    import xbmc
+    import xbmcvfs
+    import xbmcaddon
+    import xbmcplugin
+    import xbmcgui
+except ImportError:
+    from tests.stubs import xbmc, xbmcgui, xbmcaddon, xbmcplugin, xbmcvfs
+
+__addon__ = xbmcaddon.Addon()
+__scriptid__ = __addon__.getAddonInfo('id')
+__scriptname__ = __addon__.getAddonInfo('name')
+__version__ = __addon__.getAddonInfo('version')
+__language__ = __addon__.getLocalizedString
+
+__cwd__ = xbmc.translatePath(__addon__.getAddonInfo('path'))
+__profile__ = xbmc.translatePath(__addon__.getAddonInfo('profile'))
+__resource__ = xbmc.translatePath(os.path.join(__cwd__, 'resources', 'lib'))
+__temp__ = xbmc.translatePath(os.path.join(__profile__, 'temp', ''))
+
+sys.path.append(__resource__)
+
+from NapisyUtils import NapisyHelper, log, normalizeString, clean_title, parse_rls_title
+
+
+def search(item):
+    helper = NapisyHelper()
+    subtitles_list = helper.get_subtitle_list(item)
+
+    if subtitles_list:
+        for it in subtitles_list:
+            listitem = xbmcgui.ListItem(label=it["language_name"],
+                                        label2=it["filename"],
+                                        iconImage=str(it["rating"]),
```

Comment:
please remove iconImage and thumbnailImage and use only setArt

Reply:
Can u add docs for the `setArt` api?

Reply:
https://codedocs.xyz/xbmc/xbmc/group__python__xbmcgui__listitem.html#gad3f9b9befa5f3d2f4683f9957264dbbe